config/cesm/config_grids.xml

```diff
@@ -8,11 +8,11 @@
     =========================================
     The notation for the grid longname is
         a%name_l%name_oi%name_r%name_m%mask_g%name_w%name
-    where
+    where     
```

Comment:
Please avoid adding unnecessary whitespace to files, it makes the review process difficult.  Most editors can be configured to do this.